sqlalchemy_imageattach/entity.py

```diff
@@ -408,6 +420,64 @@ def _images_succeeded(cls, session):
         cls._stored_images.clear()
         cls._deleted_images.clear()
 
+    def _original_images(self, **kwargs):
+        """(:class:`Image`) A list of the original images.
```

Comment:
Although it’s a trivial docs convention, I’ve used leading `(:class:`...`) ` for attributes/properties in docstring.  As it’s not a property but a method, it’d better use [`:return:`/`:rtype` field lists][1] instead.

[1]: http://sphinx-doc.org/domains.html#info-field-lists